src/FreeAge/client/minimap.cpp

```diff
@@ -98,7 +98,7 @@ void Minimap::Update(Map* map, const std::vector<QRgb>& playerColors, QOpenGLFun
       }
     } else if (item.second->isUnit()) {
       ClientUnit* unit = AsUnit(item.second);
-      if (map->IsUnitInFogOfWar(unit)) {
+      if (unit->IsGarrisoned() || map->IsUnitInFogOfWar(unit)) {
```

Comment:
Could be combined into something like `IsVisible` if the conditions grow in number.

Reply:
That seems like a good idea.